cmake/EthCompilerSettings.cmake

```diff
@@ -52,6 +52,13 @@ if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MA
 	add_compile_options(-Wimplicit-fallthrough)
 	add_compile_options(-Wsign-conversion)
 
+	# While this should work on CMake 3.3+, it fails on Ubuntu 18
+	if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.16)
```

Comment:
```suggestion
	if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.16)
```

Reply:
All our cmake files follow this style, i'd say >95% of the time. I don't care which though.

Reply:
Oh. And I specifically looked the whole file to make sure I'm not suggesting something out of line. Well, looks like this file is an exception then :) Disregard my comment in that case.

Reply:
Yes, it seems this file is nicer, i'll change it.